[Refact] #13 FolderList, MemoList container로 변경

sewonkimm opened the pull request.

sewonkimm committed:
[Refact] #13 FolderList를 container로 변경

sewonkimm committed:
Merge branch 'feat/#13/markup' of https://github.com/sewonkimm/MojimojiMemoji into feat/#13/markup

sewonkimm committed:
[Refact] #13 MemoList를 container로 변경

soyoungjeong commented on src/containers/FolderListPage/index.tsx:
```diff
@@ -3,7 +3,7 @@ import styles from './styles.scss';
-const FolderList = () => {
+const FolderListPage = () => {
   return (
     <div className={styles.folderList}>
```
Header가 사용되지 않는 페이지도 있기 때문에 이를 App이 아닌 각 페이지에 넣어주는 것이 좋을듯 합니다!

sewonkimm replied:
수정했습니다😉

sewonkimm committed:
[Refact] #13 App container 수정

Header가 사용되지 않는 페이지가 있으므로 App에 있던 Header 컴포넌트를
FolderListPage와 MemoListPage에 넣어줌.

soyoungjeong reviewed (approved):
수고하셨습니다😊😊
